src/promptflow-devkit/promptflow/_sdk/data/docker/Dockerfile.jinja2

```diff
@@ -50,7 +50,7 @@ RUN conda run -n {{env.conda_env_name}} sh /flow/{{ env.setup_sh }}
 
 EXPOSE 8080
 
-COPY ./connections/* /connections/
+COPY ./connections /connections
```

Comment:
Why change this?

Reply:
If the connections are an empty directory, the previous method will cause Docker build to fail.